tests/e2e/support/api/http.ts

```diff
@@ -25,7 +25,7 @@ export const request = async ({
     Authorization: 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64')
   }
 
-  if (config.apiToken) {
+  if (!config.basicAuth) {
     authHeader.Authorization = 'Bearer ' + tokenEnvironment.getToken({ user }).accessToken
   }
```

Comment:
swap it like:
```js
const authHeader = {
    Authorization: 'Bearer ' + tokenEnvironment.getToken({ user }).accessToken
  }

  if (config.basicAuth) {
    authHeader.Authorization = 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64')
  }
```

Reply:
it doesn't work. if `BASIC_AUTH=true` token is undefined

Reply:
Ah, when `BASIC_AUTH=true`, we don't set `accessToken`